README.markdown

```diff
@@ -221,54 +329,118 @@ class app_widgetContainer {
 }
 ```
 
-* For functions and init methods, prefer named parameters for all arguments unless the context is very clear. Include external parameter names if it makes function calls more readable.
+### Nested structs
+
+When declaring a `struct` or `enum` within a scope **only the top level "owner" is UpperCamelCase**.
+
+The inner structs use **lowerCamelCase**.
+
+```swift
+struct Database { // UpperCamelCase
+
+    struct key { // lowerCamelCase
```

Comment:
Well isn't how the native API is made of? `thisClass.thisKindOfState.thisValue.thisFunction` ?